### ISSUE TITLE
Test Issue for using GitHub

### ISSUE THREAD

**Issue by ericrcaz** [Wireless-Innovation-Forum/Facilities#1]
This is submission of a test issue in GitHub.  Does GitHub allow attachment of documents?

**Comment by LeePucker** [comment]
You can attach a picture or a link:
https://help.github.com/articles/issue-attachments/

**Comment by LeePucker** [comment]
You can upload a picture or attach a link: https://help.github.com/articles/issue-attachments/

**Comment by LeePucker** [comment]
Note that issues can be opened and closed without intervention by the work group. I am not sure if this can be fixed

**Comment by ericrcaz** [comment]
Yes, that could be a problem, I may have been able to close since I was the initiator of the issue,

**Comment by LeePucker** [comment]
And I can reopen because I am an admin

**Comment by ericrcaz** [comment]
See Permissions level
 https://help.github.com/articles/permission-levels-for-an-organization-repository/

I believe if rights are set to "read access team" the user would have sufficient permissions to submit and review comments.

**Comment by LeePucker** [comment]
Try to edit now

**Comment by ericrcaz** [comment]
I still see the see the ability to close comment
